### PR TITLE
22353 correction of continuation in business

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#22353

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/schemas/correction.json [hunk 1]
```diff
@@ -123,7 +123,11 @@
                   "BEN",
                   "BC",
                   "CC",
-                  "ULC"
+                  "ULC",
+                  "CBEN",
+                  "C",
+                  "CCC",
+                  "CUL"
```
Comment (severinbeauvais) [inline src/registry_schemas/schemas/correction.json:130]: About about sorting these?

File: src/registry_schemas/schemas/change_of_address.json [hunk 2]
```diff
@@ -14,55 +14,49 @@
...
-            "then": {
-                "properties": {
-                    "offices": {
-                        "registeredOffice": {
-                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-                        },
-                        "recordsOffice": {
-                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    },
+                    "else": {
```
Comment (severinbeauvais) [inline src/registry_schemas/schemas/change_of_address.json:39]: Won't this accidentally require firms to have both registered and records addresses? I think they only have the 1 set.

https://dev.edit.business.bcregistry.gov.bc.ca/FM1083485/correction/?correction-id=150661
image

https://dev.edit.business.bcregistry.gov.bc.ca/BC0883553/correction/?correction-id=150662
image

Comment (vysakh-menon-aot) [reply src/registry_schemas/schemas/change_of_address.json:39]: Firms don't have changeOfAddress, its changeOfRegistration

Comment (severinbeauvais) [reply src/registry_schemas/schemas/change_of_address.json:39]: Ah, OK. I thought this was related to corrections.

Resolving.

**Review by severinbeauvais — APPROVED** [review]
LGTM. Make sure you call out what filings to verify for QA.

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**